shopBuddy/shopBuddy/ScanBarcodeViewController.m

```diff
@@ -0,0 +1,107 @@
+//
+//  ScanBarcodeViewController.m
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/14/22.
+//
+
+#import "ScanBarcodeViewController.h"
+#import "AVFoundation/AVFoundation.h"
+#import "ItemDetailViewController.h"
+
+@interface ScanBarcodeViewController ()
+@property (weak, nonatomic) IBOutlet UIView *preview;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *scanButton;
+@property (weak, nonatomic) IBOutlet UIButton *lookupButton;
+@property NSString *barcode;
```

Comment:
You can define these as ivars instead of properties. Properties are meant for public class attributes. https://aruniphoneapplication.blogspot.com/2018/09/property-vs-instance-variable-ivar-in.html